src/screens/App/screens/Dashboard/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import Dashboard from './layout'
+import { books } from '../../../../services/bookService';
+
+class DashboardContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = { books }
+    this.filter = this.filter.bind(this);
+  }
+
+  filter (e) {
```

Comment:
i like use some name by filter o selectors parameters to be more descriptive.
Something like, `value`, `option`, `selection`